commands.js

```diff
@@ -291,19 +291,25 @@ var commands = exports.commands = {
 		switch (cmd) {
 		case 'privateroom':
 			if (!this.can('makeroom')) return;
-			if (room.isOfficial) return this.sendReply('Official rooms can not be secret.');
-			if (room.isPublic) delete room.isPublic;
+			// if (room.isOfficial) return this.sendReply('Official rooms can not be secret.');
+			/*if (room.chatRoomData) {
+				delete room.chatRoomData.isPublic;
```

Comment:
Not sure why we're commenting this out, but even so, why are we doing two different types of comments line after line ?_?